sr_tactile_sensor_controller/sr_tactile_sensor.launch

```diff
@@ -1,25 +1,29 @@
 <launch>
+  <!-- prefix for the spawner node and the controller (might be different from joint_name prefix) -->
+  <arg name="prefix" default=""/>
+  <!-- config_dir to access the correct set of tactile sensors parameters (which contain the joint_name prefix) -->
+  <arg name="config_dir" default="" />
   <!-- load configuration -->
   <group if="$(optenv BIOTAC_HAND 0)">
```

Comment:
I would really prefer that these if statements depend on args (with no defaults).

Then, at a higher level (sr_edc_bimanual.launch) there will be an arg with a default value that reads the env var. e.g.:

```xml
<arg name="biotac" default="$(optenv BIOTAC_HAND 0)"/>
```

this gives us more flexibility.